src/simulation/dynamics/_GeneralModuleFiles/THRSimConfig.h

```diff
@@ -51,6 +51,8 @@ THRSimConfig
     char label[10];                                 //!< [-], label name of the TH device being simulated
     double cutoffFrequency;                         //!< [rad/s] cutoff frequency for first-order behavior
     double MaxSwirlTorque;                          //!< [Nm] Steady state magnitude of the swirl torque produced by ionic thrusters
+    std::vector<double> thrBlowDownCoeff;           //!< -- Polynomial coefficients for fuel mass to thrust blow down model in descending order
```

Comment:
If there are no units no need for `--` at the beginning of the comment

Reply:
Agreed, updating the full file to not use empty unit designations.